neurons/miner.py

```diff
@@ -299,10 +291,10 @@ async def ping(synapse: storage.protocol.Ping) -> storage.protocol.Ping:
     while True:
         try:
             # Below: Periodically update our knowledge of the network graph.
-            if step % 5 == 0:
+            if step % 300 == 0:
```

Comment:
Why are you changing this to 300, @sergioreina ?

Reply:
Sorry about that, @salahawk. I just felt like having that information pop up every five seconds was a bit too annoying in the logs, so I switched it to every five minutes. If it's causing any issues or if you prefer it the way it was, feel free to disregard it.

Reply:
> Sorry about that, @salahawk. I just felt like having that information pop up every five seconds was a bit too annoying in the logs, so I switched it to every five minutes. If it's causing any issues or if you prefer it the way it was, feel free to disregard it.

It takes dozens of minutes to benchmark - so it's going to be too long for one set-weight interval.

Reply:
Can you please revert this change and commit again? 🙏